client/src/utils/ForumContext.js

```diff
@@ -0,0 +1,32 @@
+import React, { useState, useEffect, createContext } from "react";
+import forumApi from "../utils/forum.api";
+import { filter } from 'lodash';
+
+export const ForumContext = createContext();
+
+export const ForumProvider = (props) => {
+    const [data, setData] = useState([]);
+
+    const deleteForum = (forumId) => {
```

Comment:
we should not do any operation here. context should be generic and setting data to context has to happen outside of context